src/Employer/ESFA.DAS.EmployerFeedbackEmail.Data/Repositories/EmployerFeedbackRepository.cs

```diff
@@ -11,54 +10,23 @@ namespace ESFA.DAS.ProvideFeedback.Data.Repositories
     {
-        private int _commandTimeoutSeconds = 120;
+        private const int CommandTimeoutSeconds = 120;
         private readonly IDbConnection _dbConnection;
         private const string EmployerSurveyCodes = "EmployerSurveyCodes";
-        private const string EmployerSurveyHistoryComplete = "vw_EmployerSurveyHistoryComplete";
 
         public EmployerFeedbackRepository(IDbConnection dbConnection)
         {
             _dbConnection = dbConnection;
         }
 
-        public Task<FeedbackInvite> GetLatestFeedbackInviteSentDateAsync(long feedbackId)
+        public async Task<int> GetEmployerAccountIdFromUniqueSurveyCode(Guid uniqueCode)
         {
-            var parameters = new DynamicParameters();
-            parameters.Add("@FeedbackId", feedbackId, DbType.Int64);
-            return _dbConnection.QuerySingleAsync<FeedbackInvite>(
-                commandType: CommandType.StoredProcedure,
-                sql: "[dbo].[GetLatestFeedbackInviteSentDate]",
-                param: parameters,
-                commandTimeout: _commandTimeoutSeconds);
-        }
-
-        public async Task<EmployerSurveyInvite> GetEmployerInviteForUniqueCode(Guid uniqueCode)
-        {
-            return await _dbConnection.QueryFirstOrDefaultAsync<EmployerSurveyInvite>(
-                                        $@"SELECT TOP(1) *
-                                          FROM {EmployerSurveyHistoryComplete}
-                                          WHERE UniqueSurveyCode = @{nameof(uniqueCode)}",
-                                          new { uniqueCode });
-        }
-
-        public async Task<IEnumerable<EmployerSurveyInvite>> GetEmployerUsersToBeSentInvite()
-        {
-            return await _dbConnection.QueryAsync<EmployerSurveyInvite>(
-                sql: "[dbo].[GetSurveyInvitesToSend]",
-                commandType: CommandType.StoredProcedure,
-                commandTimeout: _commandTimeoutSeconds);
-        }
-
-        public async Task<IEnumerable<EmployerSurveyInvite>> GetEmployerInvitesToBeSentReminder(int minDaysSinceInvite)
-        {
-            var minSentDate = DateTime.UtcNow.AddDays(-minDaysSinceInvite);
-            var parameters = new DynamicParameters();
-            parameters.Add("@MinSentDate", minSentDate, DbType.DateTime2);
-            return await _dbConnection.QueryAsync<EmployerSurveyInvite>(
-                sql: "[dbo].[GetSurveyRemindersToSend]",
-                param: parameters,
-                commandType: CommandType.StoredProcedure,
-                commandTimeout: _commandTimeoutSeconds);
+            return await _dbConnection.QueryFirstOrDefaultAsync<int>(
+                                        $@"SELECT AccountId
+                                        FROM EmployerSurveyCodes ESC
+                                        JOIN EmployerFeedback EF ON ESC.FeedbackId = EF.FeedbackId
+                                        WHERE ESC.UniqueSurveyCode = @{nameof( uniqueCode) }",
```

Comment:
[SonarCloud is having a hard time](https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=149&id=SkillsFundingAgency_das-provide-feedback-employer&open=AYZQplAEFUASU11F5jyQ&tab=code) with these spaces, perhaps we should remove them:

```suggestion
                                        WHERE ESC.UniqueSurveyCode = @{nameof(uniqueCode)}",
```

Reply:
@SawNawDfE this is a known issue with SonarCloud, unfortunately removing the white space won’t make a difference and it’ll still fail. Chris Barnard would override SonarCloud.

Reply:
You're right, it looks like they claimed to fix it in this issue but it's still happening: https://github.com/SonarSource/sonar-dotnet/issues/6126